src/components/Patient/PatientRegistration.tsx

```diff
@@ -0,0 +1,1050 @@
+              value={form.meta_info?.socioeconomic_status}
+              onValueChange={(value) =>
+                setForm((f) => ({
+                  ...f,
+                  meta_info: {
+                    ...(f.meta_info as any),
+                    socioeconomic_status: value,
+                  },
+                }))
+              }
+              className="flex items-center gap-4"
+            >
+              {SOCIOECONOMIC_STATUS_CHOICES.map((sec) => (
+                <>
+                  <RadioGroupItem value={sec} id={"sec_" + sec} />
+                  <Label htmlFor={"sec_" + sec}>
+                    {t(`SOCIOECONOMIC_STATUS__${sec}`)}
+                  </Label>
+                </>
+              ))}
+            </RadioGroup>
+            <br />
+            <RadioGroup
+              label={t("has_domestic_healthcare_support")}
+              value={form.meta_info?.domestic_healthcare_support}
+              onValueChange={(value) =>
+                setForm((f) => ({
+                  ...f,
+                  meta_info: {
+                    ...(f.meta_info as any),
+                    domestic_healthcare_support: value,
+                  },
+                }))
+              }
+              className="flex items-center gap-4"
+            >
+              {DOMESTIC_HEALTHCARE_SUPPORT_CHOICES.map((dhs) => (
+                <>
+                  <RadioGroupItem value={dhs} id={"dhs_" + dhs} />
+                  <Label htmlFor={"dhs_" + dhs}>
+                    {t(`DOMESTIC_HEALTHCARE_SUPPORT__${dhs}`)}
+                  </Label>
+                </>
+              ))}
+            </RadioGroup>
+          </div>
+          {/* <div id="volunteer-contact" className="mt-10">
+            <h2 className="text-lg font-semibold">
+              {t("patient__volunteer-contact")}
+            </h2>
+            <div className="text-sm">{t("volunteer_contact_detail")}</div>
+            <br />
+  
+          </div> */}
+          {/* <div id="insurance-details" className="mt-10">
+            <h2 className="text-lg font-semibold">
+              {t("patient__insurance-details")}
+            </h2>
+            <div className="text-sm">{t("insurance_details_detail")}</div>
+            <br />
+          </div> */}
+          <div className="flex justify-end mt-20">
+            <Button
+              type="submit"
+              variant={"primary"}
+              disabled={
+                patientId
+                  ? updatePatientMutation.isProcessing
+                  : createPatientMutation.isProcessing
+              }
+            >
+              {patientId ? t("save") : t("save_and_continue")}
+            </Button>
+          </div>
+        </form>
+      </div>
+      {!patientPhoneSearch.isLoading &&
+        !!duplicatePatients?.length &&
+        !!parsePhoneNumber(debouncedNumber || "") &&
+        !suppressDuplicateWarning && (
+          <DuplicatePatientDialog
```

Comment:
<img width="595" alt="image" src="https://github.com/user-attachments/assets/ff0eed8b-2c6a-4553-b70a-ecd1d684f2f3" />
Since we are updating it, why not change it to use shadcn buttons. would look prittier.

Reply:
@shivankacker Why is this closed? has this been resolved? Add a comment when you're closing a comment

Reply:
@gigincg yes resolved